src/index.ts

```diff
@@ -39,7 +40,7 @@ const client = new Client({ partials: ['MESSAGE', 'CHANNEL', 'REACTION'] });
     if (config.VERBOSE) {
       // if we are to print each message as is.
       client.on('message', (message) => {
-        console.log('>> ', message.content);
+        logger.silly('>> ', message.content);
```

Comment:
`logger.silly`? 🤔 😛

Reply:
`silly` is the lowest log level, so it wont be outputted to the console if winston is configured with **any** higher log level.